script/generate-json-schemas

```diff
@@ -3,3 +3,6 @@
 set -eu
 
 cd "$(dirname "$0")/.."
+
+$(npm bin)/ts-json-schema-generator --path ./src/inputs.ts --type RepoArray --additional-properties true --out src/json-schemas/RepoArray.json
+$(npm bin)/ts-json-schema-generator --path ./src/inputs.ts --type Instructions --additional-properties true --out src/json-schemas/Instructions.json
```

Comment:
As far as I can tell, `ts-json-schema-generator` only accepts a single `--path` argument. It won't actually error if you pass multiple, but only the last one is respected. So this means we won't be able to easily make a single schema once we need to include types from multiple files.

Just as writing this I have had a thought that we could potentially generate the schema from the `src/json-validation.ts` file, since by necessity all types will have to be imported there.

But overall I'm tempted to not try to make everything into a single schema file. I hope that having multiple schemas won't be too much of a burden. It should only require changing two files, and there are now tests that should catch when something isn't done quite right.

We can also hopefully make this change in the future if it does turn out to be simpler. Is that ok?

Reply:
Thanks for exploring this! I think it's fine to keep them as separate files, and look into generating a single file for all of them it it turns out to be simpler in the future.